.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+files: "nilearn/decomposition"
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
```

Comment:
Why not the latest version (`4.4.0`) ?

Reply:
because I copy pasted this from another file and forgot to check the latest version 🙈 

fixed now as well as updated the flake8 version